lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -824,6 +824,36 @@ DWARFASTParserClang::GetDIEClassTemplateParams(const DWARFDIE &die) {
   return {};
 }
 
+void DWARFASTParserClang::MappingDeclDIEToDefDIE(
```

Comment:
nit: Maybe you're reading this another way, but this name does not sound like a verb phrase to me (which I think it should be, as the function changes the state of the program). I'd just call it `MapDeclDIEToDefDIE`

Reply:
Done.